src/App.vue

```diff
@@ -765,6 +765,23 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
 
     this.setIsSaving(false)
   }
+
+  /**
+   * Helper to show the confirm dialogs.
+   * @param title The title content in dialog header
+   * @param message The content body
+   * @param yes The YES action label
+   * @param no The NO cancel label
+   * */
+  async showConfirmDialog (title: string, message: string, yes: string, no: string = null): Promise<any> {
+    return this.$refs.confirm.open(title, message, {
```

Comment:
Check your interface for ConfirmDialogType -- it should return boolean.

Reply:
Yep sorry, i was reverting changes here in order to push up the other commits for PR comments. 
Still wip

Reply:
Fixed!